spec/rspec/rails/fixture_support_spec.rb

```diff
@@ -13,5 +13,15 @@ module RSpec::Rails
         expect(group).to respond_to(:fixture_path=)
       end
     end
+
+    it "doesn't raise a WrongScopeError on Rails 6.1" do
```

Comment:
If the spec only needs to run on a version of Rails it should be flagged as such as we have CI running against multiple Rails and Ruby versions.